src/Solnet.Rpc/Utilities/AddressExtensions.cs

```diff
@@ -94,7 +94,7 @@ public static bool TryFindProgramAddress(IEnumerable<byte[]> seeds, byte[] progr
         /// </summary>
         /// <param name="data">The data to hash.</param>
         /// <returns>The hash.</returns>
-        private static byte[] Sha256(byte[] data)
+        public static byte[] Sha256(byte[] data)

```

Comment:
I think this should be moved into another class. Doesn't make sense to make this public here, but since its needed to be re-used, it makes sense to move it somewhere else. E.g. A `HashingUtilities` static class ?